notebooks/1-lightcurves/TIKEBook_Lesson1.ipynb

```diff
@@ -0,0 +1,1265 @@
+{
```

Comment:
<p>May want to clean up this example. Ideally, you'd want to create a mask for the aperture, and calculate the flux only within that mask, rather than over the full 20x20 array from TESSCut. It's also not clear to me that the "dips" in the light curve are transits. May want to pick a planet example where you know you will get very obvious transits from the SAP light curve that you're creating here.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/spacetelescope/project-tikebook/pull/16/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='49'/>

Reply:
<p>okay, I did show them how to do the masking earlier but I can do it here too. Will continue to look for better transit example, see next commit. </p>